R/load_flu_hosp_data.R

```diff
@@ -100,9 +109,21 @@ load_flu_hosp_data <- function(as_of = NULL,
         value = previous_day_admission_influenza_confirmed
       ) %>%
       dplyr::arrange(geo_value, date)
+    } else {
+      state_dat <- data %>%
+        dplyr::transmute(
+          geo_value = tolower(state),
+          date = date - 1,
+          epiyear = lubridate::epiyear(date),
+          epiweek = lubridate::epiweek(date),
+          value = previous_day_admission_influenza_confirmed,
```

Comment:
at a minimum, we could delete this right?

Reply:
yeah, I'm fixing that.